src/components/LogItem.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+    import { format } from 'date-fns';
+    import type { UserMessage } from '../models/user.ts';
+    import Waves from './Waves.svelte';
+    export let item: UserMessage;
+
+    const LABELS = {
+        open: 'Bypass: Valve set to open',
+        close: 'Bypass: Valve set to close',
+        bypass: 'Bypass: Bypass mode released',
+        flow: 'Closed valve due to leak detection',
+    };
+</script>
+
+<span
+    class="relative grid shrink-0 overflow-hidden rounded-2xl p-4 text-base"
+    data-ty={item.ty}
```

Comment:
Reminder: the `UserMessage.ty` field no longer exists. In fact, `UserMessage` has been removed altogether in #18.